command_builder.test.ts

```diff
@@ -13,7 +13,8 @@ Deno.test("Quick example", async () => {
 });
 
 Deno.test("xargs one-line", async () => {
-  const result = await $`echo hello`
+  const result = await $`echo ignore`
+    .$(`echo hello`)
```

Comment:
This is not xargs test.
Can you separate to other test?

Reply:
done